style.css

```diff
@@ -67,6 +75,10 @@
   justify-content: center;
 }
 
+.flex-justify-content-space-between{
```

Comment:
you can name this as justify-between as the former name is too long to type out.